backend/src/controllers/postTagController.ts

```diff
@@ -11,6 +16,12 @@ export async function getPostTags(): Promise<IPostTag[]> {
   return postTags;
 }
 
+export async function getPostTagByName(name: string): Promise<IPostTag> {
+  const postTag = await getPostTagByNameDB(name);
+  if (!postTag) throw new Error('The request post tag does not exist');
```

Comment:
We need to apply the clean code, so put the {...} on the if statement

Reply:
Yes you're right, sorry